### ISSUE TITLE
Abbreviations for column description, space reduction

### ISSUE THREAD

**Issue by matu3ba** [dalance/procs#54]
procs --version
procs 0.9.20
The column description `State` and `Nice` could be shorter and waste space.
Would it be possible to reduce the space between columns?

**Comment by dalance** [comment]
I'll add the configuration for column description like below:

```
[[columns]]
kind = "State"
header = "st"

[[columns]]
kind = "Nice"
header = "ni"
```

**Comment by dalance** [comment]
I released v0.10.0.
